### PR TITLE
Add support for multiple tags

### LINKED ISSUE #151: Allow multiple architectures, image tags for image builds
The dev container CLI will soon support a platform argument, and https://github.com/devcontainers/cli/pull/61 introduces the ability to do multiple tags.

The challenge here is when you do a multiple architecture build, you can't use the image to run locally.  This makes me wonder if this should either be a separate action, or requires an optional `runCmd` as proposed with #150.

//cc @stuartleeks

### PR DESCRIPTION
**Pull request by natescherer**
Hello!
This PR adds support for pushing multiple tags via the existing imageTag input; you can now provide a comma-separated list of tags.

I tested with the following scenarios, both for the GH Action and AzDO Task:
- Single Arch, Multi Tag
- Single Arch, Single Tag
- Single Arch, No Tag 'latest' fallback
- Multi Arch, Multi Tag
- Multi Arch, Single Tag
- Multi Arch, No Tag 'latest' fallback

This will close #151.

Let me know if you have any questions or would like me to make any changes!

### CODE REVIEW

**Comment by stuartleeks** [comment]
Thanks @natescherer! I've had a quick scan of the changes and it looks good so far.

Could you update the docs (root README for GitHub action and under azdo-task for the Azure DevOps task)?

It would also be really good to add some tests that validate that this is working (and keeps working).

**Comment by stuartleeks** [comment]
Apologies for the delay replying - thanks for updating the docs @natescherer.

Do you have any thoughts about how we could approach automated testing of the multi-tag support?

File: action.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ inputs:
     description: Image name (including registry)
   imageTag:
     required: false
-    description: Image tag (defaults to latest)
+    description: One or more comma-separated image tags (defaults to latest)
```
Comment (chrmarti) [inline action.yml:13]: Would it make sense to make it a string OR and array of strings? (To make use of YAML syntax for the list.)

**Review by chrmarti — COMMENTED** [review]
Left a comment, LGTM otherwise, thanks!

File: action.yml (hunk 1 above)
Comment (natescherer) [reply action.yml:13]: Is that possible? I thought action inputs can only be strings because they get stores as environment variables.

**Comment by natescherer** [comment]
> Apologies for the delay replying - thanks for updating the docs @natescherer.
> 
> Do you have any thoughts about how we could approach automated testing of the multi-tag support?

No worries on the delay, December and January are extremely busy for me and I haven't had much time for OSS. Here's a test I developed in my own repo that seems to work; hopefully I substituted in the right values for devcontainers/ci.

File: action.yml (hunk 1 above)
Comment (chrmarti) [reply action.yml:13]: Sounds good, I wasn't aware of the limitations here. Thanks!

**Review by chrmarti — APPROVED** [review]

**Review by stuartleeks — APPROVED** [review]
LGTM - thanks!

**Pull request merged**